docs/source/noise/row_noise.rst

```diff
@@ -89,3 +89,50 @@ parameter:
 
 When applying :code:`omit_row` noise, each row of data is selected for omission
 independently with probability :code:`row_probability`.
+
+Duplicate with guardian
+-----------------------
+
+A known entity resolution challenge is children being reported multiple
+times at different addresses. This can occur when family structures are
+complex and children might spend time at multiple households. A related
+challenge occurs with college students, who often are counted both at their
+university and at their guardian’s home address.
+
+To facilitate this type of error, we have simulants assigned to guardians
+within the simulation. Sometimes, those guardians may move and live at
```

Comment:
```suggestion
within the simulation. Sometimes, those guardians may live at
```

Reply:
Since this can happen for other reasons besides moving (e.g. all college GQ at initialization)